app/src/main/res/layout/list_group.xml

```diff
@@ -36,6 +36,7 @@
             android:paddingBottom="@dimen/text_padding_top"
             android:paddingLeft="@dimen/text_padding_top"
             android:paddingTop="@dimen/text_padding_top"
+            android:paddingRight="@dimen/text_padding_top"
```

Comment:
hey, please refactor this property `text_pading_top` with a generic name like `faq_text_padding`

Reply:
ok, I'll do it. Thanks for that